scripts/perftests/testCases/OrchardCore-5dbd92c.ps1

```diff
@@ -20,7 +20,8 @@ if ($globalJsonPath) {Remove-Item "$sourcePath\$globalJsonPath"}
 
 #########################################################
 # https://github.com/G-Research/NuPerfMonitor/issues/8
-Set-Content -Path "$sourcePath\MSBuild.rsp" -Value "-p:EnableSdkContainerSupport=false"
+# Workaround for errors due to a vulnerability scanning (https://github.com/NuGet/Home/blob/dev/proposed/2022/vulnerabilities-in-restore.md)
+New-Item "$sourcePath\Directory.Build.rsp" -Value "/p:NuGetAudit=disable"
```

Comment:
`NuGetAudit=false` ?

Reply:
Nah, it seems it needs to be `disable`, see https://github.com/NuGet/Home/blob/dev/proposed/2022/vulnerabilities-in-restore.md#enabling-vulnerability-auditing

Reply:
saw this doc https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages#disabling-security-auditing but alright 👍